src/main/java/com/github/maxopoly/MemeMana/command/CmdMana.java

```diff
@@ -0,0 +1,91 @@
+package com.github.maxopoly.MemeMana.command;
+
+import vg.civcraft.mc.civmodcore.command.PlayerCommand;
+import vg.civcraft.mc.namelayer.NameAPI;
+import com.devotedmc.ExilePearl.ExilePearlPlugin;
+import com.devotedmc.ExilePearl.ExilePearl;
+import com.github.maxopoly.MemeMana.model.MemeManaPouch;
+import com.github.maxopoly.MemeMana.model.MemeManaOwner;
+import com.github.maxopoly.MemeMana.MemeManaIdentity;
+import com.github.maxopoly.MemeMana.MemeManaPlugin;
+import org.bukkit.entity.Player;
+import org.bukkit.command.CommandSender;
+import java.util.UUID;
+import java.util.List;
+import java.util.LinkedList;
+
+public class CmdMana extends PlayerCommand {
+	public CmdMana(String name) {
+		super(name);
+		setIdentifier("mana");
+		setDescription("The base command for Meme Mana");
+		setUsage("\n/mana show -- Show your current mana\n/mana refill [Amount] -- Refill the pearl in your hand by an amount\n/mana transfer <Player> <Amount> -- Transfer some mana to another player");
+		setArguments(1,3);
+	}
+
+	public boolean execute(CommandSender sender, String [] args) {
+		if (args[0] == "show") {
```

Comment:
I'm tempted to complain about the fact that you didnt do a switch case here, but the compiler would turn it into the exact same thing as yours, so you win today.